gradle/metalava.gradle

```diff
@@ -6,18 +6,18 @@ dependencies {
     // Metalava isn't released yet. Check in its jar and explicitly track its transitive deps.
     metalava files('../metalava/metalava.jar')
 
-    metalava "com.android.tools.external.org-jetbrains:uast:27.2.0-alpha11"
-    metalava "com.android.tools.external.com-intellij:kotlin-compiler:27.2.0-alpha11"
-    metalava "com.android.tools.external.com-intellij:intellij-core:27.2.0-alpha11"
-    metalava "com.android.tools.lint:lint-api:27.2.0-alpha11"
-    metalava "com.android.tools.lint:lint-checks:27.2.0-alpha11"
-    metalava "com.android.tools.lint:lint-gradle:27.2.0-alpha11"
-    metalava "com.android.tools.lint:lint:27.2.0-alpha11"
-    metalava "com.android.tools:common:27.2.0-alpha11"
-    metalava "com.android.tools:sdk-common:27.2.0-alpha11"
-    metalava "com.android.tools:sdklib:27.2.0-alpha11"
-    metalava "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.20"
-    metalava "org.jetbrains.kotlin:kotlin-reflect:1.4.20"
+    metalava "com.android.tools.external.org-jetbrains:uast:30.0.0-alpha14"
```

Comment:
Yes I was unsure if we should update this or not. Since metalava was built with the previous major versions of dependencies - could face the issues, I guess.

Reply:
IMO if there's not explicit problems with the current ones - it's better to keep them

Reply:
in https://github.com/mapbox/mapbox-maps-android/pull/1167, I rebuilt metalava from source with these dependencies, I just forgot to update these dependencies in the gradle file 😅 .. These dependencies are updated in the README. 

We should use the matching dependencies where the metalava jar is built.